.gitignore

```diff
@@ -12,3 +12,4 @@ docs_rst/_build
 build/*
 *.egg-info
 dist/*
+.vscode
```

Comment:
this should probably go in your global `gitignore`

Reply:
IMO if our code doe not do anything with VS code then it should go in here, right?  That's one less step for everyone that's working on the code.

Reply:
I use VS Code too. I put that line in my global `gitignore` so I don't have to add it to every project. Overall less work I think but definitely fine to leave it in. :)